src/main/java/build/buf/protovalidate/internal/evaluator/AnyEvaluator.java

```diff
@@ -34,7 +34,8 @@
  * com.google.protobuf.Any}'s within an expression, breaking evaluation if the type is unknown at
  * runtime.
  */
-class AnyEvaluator extends EvaluatorBase implements Evaluator {
+class AnyEvaluator implements Evaluator {
+  private final ConstraintViolationHelper constraintViolationHelper;
```

Comment:
nit: I would call the field `helper` (here and throughout) to make the call sites more readable. They are so verbose, it's a bit of a chore to read or scan the code with this long name.